### ISSUE TITLE
Watch does not receive notifications (even though it's connected)

### ISSUE THREAD

**Issue by themelispan** [Bellafaire/Android-Companion-App-For-BLE-Devices#5]
Hi, I've managed to get the watch connected with the phone (The square on the top right is green and I've granted notification permission). The phone says that it sends the data (the weird thing is, the status is nothing, see the picture)
![Screenshot_2021-10-02-17-48-31-473_com example smartwatchcompanionapp](https://user-images.githubusercontent.com/49806838/135721741-2b1312f7-1cd9-4252-a19f-8ff97b831f45.jpg)

This is what appeared on the Serial Monitor
![Screenshot from 2021-10-02 17-59-30](https://user-images.githubusercontent.com/49806838/135722099-dda32c0d-c735-4f50-b66e-831d5cc053a4.png)

The watch does not receive any notifications and also, for some reason the bottom right button on home screen does nothing.

I would appreciate some help

**Comment by Bellafaire** [comment]
Hello, I see that you're using the older version of the app. Does the watch update it's time correctly after connecting? 
The notifications aren't showing due to a permissions issue. Normally the 'grant notification permissions' button would automatically take you to the settings but it seems to be broken. You should be able to manually enable this by going to your settings and searching 'notification access' then manually granting the permission to the app. After restarting the app it should have no problem displaying notifications. 

Let me know if this solves your issue.

**Comment by themelispan** [comment]
The watch updates the time correctly after connecting. I've double-checked that I have granted notification permission to the app. You said that I am using an older version of the app. Is there a newer version that is supported by the watch firmware?

**Comment by Bellafaire** [comment]
Unforuntately at the moment the t-watch firmware is only compatible with this version. I haven't had time to update it due to other projects. If the permission is granted try force stopping the app and restarting it.

**Comment by themelispan** [comment]
I do not think it that worked. for some reasson my phone does not receive notifications for the links I send from the PC anymore so I can't really make sure. Is there anything else I can try?

**Comment by Bellafaire** [comment]
If you compile the app yourself and upload it to your phone through android studios you'll be able to get a stack trace which would give a better idea. Unfortunately without that I can't be sure what the issue is.
